### PR TITLE
Adding init to fix zombie process in builder and gameplay

### PR DESCRIPTION
**Pull request by laheau**

### CODE REVIEW

File: workers/gameplay/src/bots.rs [hunk 1]
```diff
@@ -397,9 +488,12 @@ impl Game {
         ))
     }
 }
-
 impl Drop for Game {
     fn drop(&mut self) {
+        unsafe {
+            let _ = kill(-(self.bot_a.id().unwrap_or_default() as i32), 18);
+            let _ = kill(-(self.bot_b.id().unwrap_or_default() as i32), 18);
+        };
```
Comment (alexwaeseperlman) [inline workers/gameplay/src/bots.rs:496]: I thought the `id()` method returns pid, and `kill(-x)` assumes `x` is a gpid

Comment (laheau) [reply workers/gameplay/src/bots.rs:496]: Yes but i set the gpid to the id when running `Command::new`. ` .process_group(0)` sets the gpid of the child to it's own pid, and it's children inherits it.

Comment (alexwaeseperlman) [reply workers/gameplay/src/bots.rs:496]: I see, that's cool

**Comment by alexwaeseperlman** [comment]
Added init to docker-compose on another branch

**Pull request closed without merging**